### DISCUSSION TITLE
Including more content in the OpenAPI 3 specs

### DISCUSSION THREAD

**Discussion by jonerickson** [tailflow/laravel-orion#284] [Ideas]
Hi there, are there any plans to provide ways to add more content to the generated spec? For example, being able to add descriptions to path operations would be great. If there is no plan, I would be interested in opening a PR to help out. My current thought would be to include attributes in the controllers to help define those descriptions. I know you already support customizing tags in the config, but it might be a little hard to do that to when it comes to all the possible operations. Any recommendations youd go about it?

**Comment by alexzarbn** [comment]
Hi @JonErickson,

No plans for now to add more content to the generated specs. It would be interesting to see your approach with attributes though!

Also, you could try generating specs, then adding content to the `specs.yaml` or `specs.json` file manually and continue regenerating specs - most of the things won't be overwritten.
